src/settings.py

```diff
@@ -217,6 +217,7 @@ def trigger_post_change_hook(setting_name, old_value, new_value):
 @post_change_hook("nginx_compatibility")
 @post_change_hook("webadmin_allowlist_enabled")
 @post_change_hook("webadmin_allowlist")
+@post_change_hook("dns_exposure")
 def reconfigure_nginx(setting_name, old_value, new_value):
     if old_value != new_value:
         regen_conf(names=["nginx"])
```

Comment:
Hmm this triggers the regenconf but I see no change in the in the nginx conf template

Also if the intention is to disable ipv6 on nginx/postfix, maybe we should rename the setting name `dns_exposure` to something like `network_stack` or idk